### PR TITLE
✨ Source Sendgrid : Migrate to Manifest-only

### PR DESCRIPTION
**Pull request by topefolorunso**

### CODE REVIEW

**Review by natikgadzhi — APPROVED** [review]
Manually ran regression tests, they're saying they're failing on schema diffs, but that's a blanket problem we need to fix, not this PR specific (cc @alafanechere). 

We can ship this!

File: airbyte-integrations/connectors/source-sendgrid/metadata.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ data:
   name: Sendgrid
   remoteRegistries:
     pypi:
-      enabled: true
+      enabled: false
```
Comment (artem1205) [inline airbyte-integrations/connectors/source-sendgrid/metadata.yaml:28]: Why do we want to disable pypi version?

File: airbyte-integrations/connectors/source-sendgrid/source_sendgrid/config_migrations.py
Comment (artem1205) [inline airbyte-integrations/connectors/source-sendgrid/source_sendgrid/config_migrations.py:1]: How will our customers migrate from older versions ?

**Review by artem1205 — COMMENTED** [review]
@natikgadzhi , left some comments below, could you please take a look?

**Comment by natikgadzhi** [comment]
PyPI is intentional, manifest-only sources work in pyairbyte out of the box, and do not ship their own Python package. It's a good idea to clear the older versions out, or at least upload a readme to pypi docs in the future. 

Config I assume should be non-breaking for _this_ version. People on older versions are effed. They'd have to setup a new connection and resync.

**Pull request merged**